backend/apps/events/tests/test_event_rudview.py

```diff
@@ -0,0 +1,65 @@
+from apps.accounts.factories import ProfileFactory
+from apps.events.factories import EventFactory, EventTypeFactory
+from django.test import TestCase
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+
+class TestEventsListView(TestCase):
```

Comment:
I guess, this one must be a TestRetrieveUpdateDestroyEventView

Reply:
Oops... You are right, thank you.